app-backend/db/src/main/scala/HITLJobDao.scala

```diff
@@ -0,0 +1,99 @@
+package com.rasterfoundry.database
+
+import com.rasterfoundry.database.Implicits._
+import com.rasterfoundry.datamodel.PageRequest
+import com.rasterfoundry.datamodel._
+
+import doobie._
+import doobie.implicits._
+import doobie.implicits.javasql._
+import doobie.postgres.implicits._
+
+import java.util.UUID
+
+object HITLJobDao extends Dao[HITLJob] {
+  val tableName = "hitl_jobs"
+
+  val selectF: Fragment = sql"""
+      SELECT
+        id, created_at, created_by, modified_at, owner,
+        campaign_id, project_id, status, version
+      FROM
+    """ ++ tableF
+
+  override val fieldNames = List(
+    "id",
+    "created_at",
+    "created_by",
+    "modified_at",
+    "owner",
+    "campaign_id",
+    "project_id",
+    "status",
+    "version"
+  )
+
+  def unsafeGetById(id: UUID): ConnectionIO[HITLJob] =
+    query.filter(id).select
+
+  def getById(id: UUID): ConnectionIO[Option[HITLJob]] =
+    query.filter(id).selectOption
+
+  def list(
+      page: PageRequest,
+      params: HITLJobQueryParameters,
+      user: User
+  ): ConnectionIO[PaginatedResponse[HITLJob]] =
+    query
+      .filter(params)
+      .filter(user.isSuperuser && user.isActive match {
+        case true  => fr""
+        case false => fr"owner = ${user.id}"
+      })
+      .page(page)
+
+  def getNewVersion(
+      user: User,
+      campaignId: UUID,
+      projectId: UUID
+  ): ConnectionIO[Int] =
+    query
+      .filter(fr"owner = ${user.id}")
+      .filter(fr"campaign_id = ${campaignId}")
+      .filter(fr"project_id = ${projectId}")
+      .list
+      .map { _.size }
```

Comment:
This will be problematic in a case in which one is deleted. This could lead to duplicate version ids